test/generators/whenever_install_generator_test.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "generators/boring/whenever/install/install_generator"
+
+class WheneverInstallGeneratorTest < Rails::Generators::TestCase
+  tests Boring::Whenever::InstallGenerator
+  setup :build_app
+  teardown :teardown_app
+
+  include GeneratorHelper
+  include ActiveSupport::Testing::Isolation
+
+  def destination_root
+    app_path
+  end
+
+  def test_should_install_whenever_gem
```

Comment:
Since test steps are simple, I think we can merge test_should_install_whenever_gem and test_should_generate_schedule_file to "test_should_configure_whenever_gem" and add the following content:

```
def test_should_configure_whenever_gem
    Dir.chdir(app_path) do
      quietly { run_generator }

      assert_gem "whenever"
      assert_file "config/schedule.rb"
    end
end
```